TwitchLib.EventSub.Core/SubscriptionTypes/Channel/ChannelSuspiciousUserMessage.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using TwitchLib.EventSub.Core.Models.ChannelSuspiciousUser;
+
+namespace TwitchLib.EventSub.Core.SubscriptionTypes.Channel
+{
+    public sealed class ChannelSuspiciousUserMessage : ChannelSuspiciousUserBase
+    {
+        /// <summary>
+        /// A list of channel IDs where the suspicious user is also banned.
+        /// </summary>
+        public string[] SharedBanChannelIds { get; set; } = [];
+        /// <summary>
+        /// User types (if any) that apply to the suspicious user, can be “manual”, “ban_evader_detector”, or “shared_channel_ban”.
+        /// </summary>
+        public string[] Types { get; set; } = [];
+        /// <summary>
+        /// A ban evasion likelihood value (if any) that as been applied to the user automatically by Twitch, can be “unknown”, “possible”, or “likely”.
+        /// </summary>
+        public string BanEvasionEvaluation { get; set; } = string.Empty;
+        /// <summary>
+        /// The structured chat message.
+        /// </summary>
+        public SuspiciousUserMessage Message { get; set; } = new SuspiciousUserMessage();
+
+        public sealed class SuspiciousUserMessage
```

Comment:
why isn't the class in its own file?

Reply:
It seemed like it wasn't a pattern in the directory, but i didn't check super thoroughly. Will break these out.